templates/home.html

```diff
@@ -80,25 +80,25 @@ <h3 class="how-title">HOW IT WORKS FOR ARTISTS</h3>
             <div class="icons">
                 <i class="fa fa-music fa-4x icon-padding" aria-hidden="true"></i>
                 <h3>RAISE</h3>
-                <p class="how-extra">RAISE MONEY TO CREATE YOUR MUSIC BY HAVING FANS INVEST IN YOUR PROJECT</p>
+                <p class="how-extra">RAISE MONEY BY GIVING PEOPLE AN INCENTIVE TO INVEST IN YOUR MUSIC</p>
             </div>
             <div class="icons">
-                <i class="fa fa-percent fa-4x icon-padding" aria-hidden="true"></i>
-                <h3>GIVE</h3>
-                <p class="how-extra">GIVE A PERCENTAGE OF SALES BACK TO PEOPLE WHO INVEST IN YOUR MUSIC PROJECT</p>
+                <i class="fa fa-money fa-4x icon-padding" aria-hidden="true"></i>
+                <h3>SPEND</h3>
+                <p class="how-extra">HIRE A TEAM OF PEOPLE TO HELP CREATE AND PROMOTE YOUR MUSIC</p>
             </div>
             <div class="icons">
                 <i class="fa fa-dollar fa-4x icon-padding" aria-hidden="true"></i>
                 <h3>EARN</h3>
-                <p class="how-extra">EARN MONEY BY BUILDING A TEAM OF PEOPLE ALL PUSHING FOR YOUR SUCCESS</p>
+                <p class="how-extra">EARN MONEY FOR YOUR MUSIC THROUGH ITUNES, SPOTIFY, AND MORE</p>
             </div>
         </div>
 
         <div class="break">
             <hr>
         </div>
         <div class="home-boxes">
-            <a class="button text-center box-margin view-artists-box" href="{% url 'artists' %}?campaign-status=Active">START INVESTING</a>
+            <a class="button text-center box-margin view-artists-box" href="{% url 'artists' %}">DISCOVER ARTISTS</a>
```

Comment:
Ohh yeah. I noticed that on a lot of Google's sites they have the same link a the top and the bottom so thought it might make sense to have have it be the same for now since there are only a few bands on there. I didn't put too much thought into this decision either so i'm cool with changing it back if you think it's better that way.